main.go

```diff
@@ -106,21 +102,17 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.BoolVar(&cfg.EnableProfiler, "enable-profiler", false, "Set if pouchd setup profiler")
 	flagSet.StringVar(&cfg.Pidfile, "pidfile", "/var/run/pouch.pid", "Save daemon pid")
 	flagSet.IntVar(&cfg.OOMScoreAdjust, "oom-score-adj", -500, "Set the oom_score_adj for the daemon")
-}
+	cfg.Runtimes = make(map[string]types.Runtime)
```

Comment:
Since we want get pointer of `cfg.Runtime` and cfg.Runtime should have been initialized, or use secondary pointer is ok, but seems also need one more line `xx = &cfg.Runtimes`, `NewRuntime(&xx)`